cms/functions.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+/* Doc Block
+ * Returns all project titles from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project titles in the arrays within the arrays pulled from the database.
+ */
+function returnProjectTitles(array $array):string {
+    $titles = '';
+    foreach ($array as $value) {
+        $titles .=$value['projectTitle']."\n";
+    }
+    return $titles;
+}
+
+/* Doc Block
+ * Returns all project title links from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project title links in the arrays within the arrays pulled from the database.
+ */
+function returnProjectTitleLinks(array $array):string {
```

Comment:
this is exactly the same as the function above - delete one of these and just have one function

Reply:
Added one function for whole form with all default values.